src/cli.js

```diff
@@ -65,71 +69,86 @@ function handleError (error) {
   process.emit('SIGINT')
 }
 
-function parseDocuments (data, primary) {
-  if (primary && !JSON.parse(data)[primary]) {
-    throw new Error(`Is the primary key correct?\n\n${primary}`)
-  }
-
-  try {
-    return primary ? JSON.parse(data)[primary] : JSON.parse(data)
-  } catch (err) {
-    try {
-      return data
-        .toString()
-        .trim()
-        .split('\n')
-        .map(JSON.parse)
-    } catch (error) {
-      throw new Error(`Is the input valid JSON/NDJSON?\n\n${error}`)
-    }
-  }
-}
-
-function check ({ query, file, url, stdIn }) {
+function check ({ query, inputFormat, outputFormat }) {
   if (!query) {
     throw Error(
       chalk.yellow(
         'You must add a query. To learn more, run\n\n  $ groq --help'
       )
     )
   }
-  if (!file && !url && !stdIn) {
-    throw Error(
-      chalk.yellow(
-        'There’s no data to query. To learn more, run\n\n  $ groq --help'
-      )
-    )
+
+  if (!/^(json|ndjson|null)$/.test(inputFormat)) {
+    throw Error(chalk.yellow(`Unknown input format: ${inputFormat}`))
```

Comment:
```suggestion
    throw Error(chalk.yellow(`Unknown input format: ${inputFormat}. Valid input formats are "json" and "ndjson".`))
```

We should probably add a constant for `VALID_INPUT` formats (DRY). Perhaps we want to support CSV and XML in the future 😆.

Reply:
I didn't add a constant, but I did separate the choice validation into its own function.